admin/templates/applications/view-application.html

```diff
@@ -197,6 +215,13 @@ <h1 class="govuk-heading-l">
                         value=(applicationData['submitAndPay']['method'])
                     ) }}
 
+                    {% if applicationData['submitAndPay']['method'] == 'Online' %}
+                        {{ summary_list_item(
+                            title='Payment reference',
+                            value=(payment_details['reference'])
+                        ) }}
+                    {% endif %}
+
```

Comment:
Same again here

Reply:
I've removed all references to `payment_details`